### PR TITLE
Addmahal

### PR DESCRIPTION
**Pull request by btracey**

### CODE REVIEW

File: statmat_test.go [hunk 1]
```diff
@@ -262,6 +262,36 @@ func TestCorrCov(t *testing.T) {
...
+					0.8, 0.3, 0.1,
+					0.3, 0.7, -0.1,
+					0.1, -0.1, 7}),
+			ans: 1.9251757377680914,
+		},
+	} {
+		var chol mat64.Cholesky
+		ok := chol.Factorize(test.Sigma)
+		if !ok {
+			panic("bad test")
```
Comment (sbinet) [inline statmat_test.go]: use `t.Fatal` instead?

Comment (kortschak) [inline statmat_test.go]: This is an internal panic, not a test panic.

**Comment by btracey** [comment]
Ping?

**Comment by btracey** [comment]
It is the same.

**Pull request merged**